### PR TITLE
[PLAT-381] Switch some packages from mocha to jest

### PR DESCRIPTION
**Pull request by drtyh2o**
## 📦 Impacts:

- [x] `mds-repository`
- [x] `mds-service-helpers`
- [x] `mds-jurisdiction-api`
- [x] `mds-compliance-batch-processor`
- [x] `mds-types`  
- [x] **BONUS**: `mds-db` (because I really want to be able to drop all _`mocha`_ dependencies)

### CODE REVIEW

**Review by nessur — APPROVED** [review]
nice!

File: .vscode/launch.json [hunk 1]
```diff
@@ -627,14 +627,12 @@
...
-        "-r",
-        "ts-node/register",
-        "--timeout",
-        "0"
+      "runtimeArgs": [
+        "--inspect-brk",
+        "${workspaceRoot}/node_modules/jest/bin/jest",
+        "--runInBand",
+        "--coverage",
+        "false"
```
Comment (PeterSP) [inline .vscode/launch.json:635]: 👍

File: .vscode/launch.json [hunk 2]
```diff
@@ -735,14 +733,12 @@
...
```
Comment (PeterSP) [inline .vscode/launch.json:741]: Why no coverage here? Would it be too low to be worth turning it on / setting a value in jest config?

File: .vscode/launch.json [hunk 3]
```diff
@@ -861,14 +857,12 @@
...
```
Comment (PeterSP) [inline .vscode/launch.json:865]: Why no coverage here? Would it be too low to be worth turning it on / setting a value in jest config?

Comment (PeterSP) [reply .vscode/launch.json:865]: Is it because it's for VSCode? IN that case ignore these comments.

Comment (drtyh2o) [reply .vscode/launch.json:865]: These are VS Code debugger launch configurations. Probably not interested in coverage here. Coverage still performed for running tests outside of the debugger.

**Review by avatarneil — APPROVED** [review]

**Pull request merged**